File: httpsnippet/src/test/resources/output/java/jsoup/full.java
String response = Jsoup.connect("http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value")
	.method(Method.POST)
	.header("Cookie", "foo=bar; bar=baz")
	.header("content-type", "application/x-www-form-urlencoded")
	.header("accept", "application/json")
	.execute().body();

File: httpsnippet/src/test/resources/output/javascript/fetch/full.js
const details ={"foo":"bar"};
const form = Object.entries(details)
    .map(([key, value]) => encodeURIComponent(key) + '=' + encodeURIComponent(value))
    .join('&')
const fetchOptions = {
  "mode": "cors",
  "method": "POST",
  "headers": {
    "Cookie": "foo=bar; bar=baz",
    "content-type": "application/x-www-form-urlencoded",
    "accept": "application/json"
  },
  "body": form
}

fetch("http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value", fetchOptions)
 .then(response => response.json())
 .then(data => console.log(data))
 .catch(error => console.log(error));